docs/data/sql_functions.yml

```diff
@@ -652,19 +652,23 @@ json:
       behavior for `ON EMPTY`, and raises an error for `ON ERROR`.
 
       ```
-      // STRING: "true"
+      -- "true"
       JSON_VALUE('{"a": true}', '$.a')
 
-      // BOOLEAN: true
+      -- TRUE
       JSON_VALUE('{"a": true}', '$.a' RETURNING BOOLEAN)
 
-      // STRING: "false"
+      -- "false"
       JSON_VALUE('{"a": true}', 'lax $.b'
           DEFAULT FALSE ON EMPTY)
 
-      // STRING: "false"
+      -- "false"
       JSON_VALUE('{"a": true}', 'strict $.b'
           DEFAULT FALSE ON ERROR)
+
+      -- 0.998D
+      JSON_VALUE('{"a.b": [0.998,0.996]}','$.[''a.b''][0]' 
```

Comment:
I was actually just thinking: doesn't using double quotes work, too, and would be easier than having to escape the single quotes?

```
JSON_VALUE('{"a.b": [0.998,0.996]}','$.["a.b"][0]' 
```

Reply:
Double quotation marks to get the correct result. If I was a developer unfamiliar with SQL, I might think double quotes are better understood.

Reply:
I think even SQL-savy people would expect this here, because this is a JSON function, and in JSON you use double quotes. So double quotes actually kind of make more sense to me than trying to use single quotes.

Reply:
so, in these examples, should we use double quotes instead?

Reply:
I think it would be the better alternative to suggest, so let's change it, yes.